packages/backend/server/src/app.module.ts

```
@@ -199,8 +204,10 @@ export function buildAppModule(env: Env) {
     )
     // doc service only
     .useIf(() => env.flavors.doc, DocServiceModule)
-    // self hosted server only
+    // worker for and self-hosted API only for self-host and local development only
     .useIf(() => env.dev || env.selfhosted, WorkerModule, SelfhostModule)
+    // static frontend routes for front flavor
+    .useIf(() => env.flavors.front, StaticFileModule)
```

Biome (2.3.13)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

This is the call path until the hook.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)